src/main/java/it/gov/pagopa/pu/sil/endpoint/PuForOrganizationReconciliationEndpoint.java

```diff
@@ -124,4 +161,23 @@ public PivotSILAutorizzaImportFlussoRTRisposta pivotSILAutorizzaImportFlussoRT(
       PivotSILAutorizzaImportFlussoRTRisposta::setFault
     );
   }
+
+  private Function<Exception, PivotSILAutorizzaImportFlussoTesoreriaRisposta> handleIngestionFlowFileTypeValidationException() {
+    return (Exception e) -> {
+      if (e instanceof IngestionFlowFileTypeValidationException ie) {
+        return FaultUtils.setFaultOnResponse(
+          new PivotSILAutorizzaImportFlussoTesoreriaRisposta(),
+          SilFaults.PIVOT_TIPO_FLUSSO_NON_VALIDO,
+          ie.getMessage()
+        );
+      }
+      return FaultUtils.unauthorizedExceptionHandler(
+        new PivotSILAutorizzaImportFlussoTesoreriaRisposta(),
+        PivotSILAutorizzaImportFlussoTesoreriaRisposta::setFault,
+        FaultBean::new,
+        SilFaults.PIVOT_ENTE_NON_VALIDO,
+        SilFaults.PIVOT_SYSTEM_ERROR
+      ).apply(e);
+    };
+  }
```

Comment:
```suggestion
    private PivotSILAutorizzaImportFlussoTesoreriaRisposta handleIngestionFlowFileTypeValidationException(Exception e) {
      if (e instanceof IngestionFlowFileTypeValidationException ie) {
        return FaultUtils.setFaultOnResponse(
          new PivotSILAutorizzaImportFlussoTesoreriaRisposta(),
          SilFaults.PIVOT_TIPO_FLUSSO_NON_VALIDO,
          ie.getMessage()
        );
      }
      return FaultUtils.unauthorizedExceptionHandler(
        new PivotSILAutorizzaImportFlussoTesoreriaRisposta(),
        PivotSILAutorizzaImportFlussoTesoreriaRisposta::setFault,
        FaultBean::new,
        SilFaults.PIVOT_ENTE_NON_VALIDO,
        SilFaults.PIVOT_SYSTEM_ERROR
      ).apply(e);
  }
```

then directly reference this method on the RegistryLogger invocation
`this::handleIngestionFlowFileTypeValidationException `